bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Executor.scala

```diff
@@ -0,0 +1,46 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.spModel.config2.{Config, ConfigSequence, ItemKey}
+import edu.gemini.spModel.obscomp.InstConstants.INSTRUMENT_NAME_PROP
+import edu.gemini.spModel.seqcomp.SeqConfigNames.INSTRUMENT_KEY
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Future, Promise}
+
+/**
+ * Created by jluhrs on 4/28/15.
+ */
+sealed class Executor(sequenceConfig: ConfigSequence) {
+
+  def run = runSequence(sequenceConfig.getAllSteps.toList)
+
+  private def runSequence(config: List[Config]): Future[Int] = {
```

Comment:
Yes, although it is a placeholder for the step execution result. The details of such artifact will reveal themselves in future sprints.